util/start_test

```diff
@@ -1512,6 +1508,25 @@ class FileHandlerWithException(logging.FileHandler):
         logging.FileHandler.handleError(self, record)
         raise OSError("fatal error, logging.FileHandler")
 
+class CommandError(Exception):
```

Comment:
Was `CalledProcessError` not available in Python 2.6 `subprocess`?

Reply:
It was, but the output argument was not added until 2.7 (it was added for check_output.) I'd prefer to leave this as-is